roles/base/templates/docker_facts.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+dver=$(sudo docker version | awk '/Version/{print $2; exit 0}')
+exper=$(sudo docker version | awk '/Experiment/{print $2; exit 0}')
```

Comment:
I'm guessing something like this will work: https://gist.github.com/erikh/61eb4216c17cb842c4f7